### PR TITLE
chore: Follow up to use global mig project in tests

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Follow up to use global mig project in tests

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [x] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: internal/service/networkcontainer/resource_network_container_migration_test.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 
 func TestMigNetworkContainer_basicAWS(t *testing.T) {
 	var (
-		projectID = acc.ProjectIDExecution(t)
+		projectID = acc.ProjectIDExecution(t) // No mig.ProjectIDGlobal because network container
```
Comment (oarbusi) [inline internal/service/networkcontainer/resource_network_container_migration_test.go:16]: `because network container` is quite vague, could we maybe rephrase it?

File: internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_migration_test.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 
 func TestMigNetworkPrivatelinkEndpointServiceDataFederationOnlineArchive_basic(t *testing.T) {
 	var (
-		projectID  = acc.ProjectIDExecution(t)
+		projectID  = acc.ProjectIDExecution(t) // No mig.ProjectIDGlobal because private link endpoint
```
Comment (oarbusi) [inline internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_migration_test.go:14]: same here

File: internal/service/networkcontainer/resource_network_container_migration_test.go (hunk 1 above)
Comment (lantoli) [reply internal/service/networkcontainer/resource_network_container_migration_test.go:16]: there can be only one container per project and provider / region.   I wanted to find a balance of giving some clues explaining why we're not using the "expected" way but not being very verbose  explaining the details of how Atlas works.

**Review by oarbusi — APPROVED** [review]

File: internal/service/networkcontainer/resource_network_container_migration_test.go (hunk 1 above)
Comment (lantoli) [reply internal/service/networkcontainer/resource_network_container_migration_test.go:16]: as discussed offline we'll leave in this way
